src/configuration.ts

```diff
@@ -405,6 +405,11 @@ export interface IChromeLaunchConfiguration extends IChromeBaseConfiguration {
    * with your default user profile.
    */
   userDataDir: string | boolean;
+
+  /**
+   * The debug adapter is running elevated. Launch Chrome unelevated to avoid the security restrictions of running Chrome elevated
+   */
+  launchUnelevatedFlag?: boolean;
```

Comment:
nit: `flag` in the naming is redundant since it's typed as a boolean, `launchUnelevated?: boolean` is good

Reply:
Fixed